src/library/scala/util/Properties.scala

```diff
@@ -13,6 +13,28 @@ package util
 import java.io.{ IOException, PrintWriter }
 import java.util.jar.Attributes.{ Name => AttributeName }
 
+sealed abstract class JavaSpecVersion extends Ordered[JavaSpecVersion] {
```

Comment:
Too much machinery to compare two integers.  That's all that's really happening.